Bixal/move.mil
# Error box on locator map (and other message boxes) cleaned up

This PR completes the fix for Pivotal bug [158978968 - error message in locator map tool looks funky](https://www.pivotaltracker.com/story/show/158978968). It might be worth refactoring the formatting of these boxes in general at some future time, but this at least patches an acute issue with single-line messages for now.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Edits the rendered markup inside a Locator map code component to match that needed for warning/callout boxes to appear as expected.
- Those boxes, however, don't appear great when the message content is just a single line, so I added a global css patch to address it (just setting a min-height on the box).

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Compile theme assets and rebuild tools
1. Go to the locator map page
1. Click "Search By Your Location"
1. DENY browser access to your location
1. See the message that appears below. It should look reasonably not-broken.

## Screenshots

Live broken:
<img width="986" alt="error message ugly - live unfixed" src="https://user-images.githubusercontent.com/29130580/43482773-a77a689c-94d7-11e8-800d-75662ae52ff4.png">


Local fixed:
<img width="981" alt="error message alignment within locator tool - local fixed" src="https://user-images.githubusercontent.com/29130580/43482730-7f28f7dc-94d7-11e8-97a9-4b4b5214f5b3.png">
